src/test/ui/consts/const-eval/heap/alloc_intrinsic_nontransient_fail.rs

```diff
@@ -0,0 +1,20 @@
+// run-pass
```

Comment:
this test is exactly the same as the test above

Reply:
Oops. Should be fixed now